app/Http/Middleware/TrimStrings.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Illuminate\Foundation\Http\Middleware\TrimStrings as Middleware;
+
+class TrimStrings extends Middleware
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing doc comment for class TrimStrings](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)